examples/fair_prep_results.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+os.chdir("..")
```

Comment:
This should not be here. Make sure that the file can be run from the root dir of the repository